src/main/java/org/apache/maven/shared/utils/PropertyUtils.java

```diff
@@ -89,22 +89,21 @@ public static Properties loadProperties( @Nonnull File file )
 
     /**
      * @param is {@link InputStream}
-     * @return The loaded properties.
-     * @deprecated As of 3.1.0, please use method {@link #loadOptionalProperties(java.io.InputStream)}. This method
+     * @return the loaded properties
+     * @deprecated use {@link #loadOptionalProperties(java.io.InputStream)} instead. This method
      *             should not be used as it suppresses exceptions silently when loading properties fails.
      */
     @Deprecated
     public static Properties loadProperties( @Nullable InputStream is )
     {
         try
         {
-            // to make this the same behaviour as the others we should really return null on any error
             Properties result = new Properties();
             if ( is != null )
             {
-                try
+                try ( InputStream in = is )
```

Comment:
I don't agree with this because have not opened this stream and it is not our task to close it. The one who opens the stream must close it.

Reply:
That makes sense. It is a behavior change though. Previously the stream was closed  in lines 120-123. This PR just changes how it's closed to avoid a now deprecated class. If we stop closing it here (and in other methods) then that might be worth a separate bug and PR that shows up in the release notes. What do you think?

Reply:
OK, leave as is and file a bug about the incorrect behavior. This should then be fixed in a subsequent release. Allso add documenation here, that behavior is wrong.

Reply:
done: https://issues.apache.org/jira/browse/MSHARED-895